cherrypicks_test.go

```diff
@@ -394,11 +394,60 @@ func TestParseCherryTargetBranches(t *testing.T) {
 		* 1.2.x\r`,
 			expected: "1.2.x",
 		},
+		"multiple branches": {
+			body: `cherry-pick to:
+		* master
+		* hosted
+		* example-branch`,
+			expected: "master hosted example-branch",
+		},
+	}
+
+	for name, test := range tests {
+		t.Log(name)
+		res, _ := parseCherryTargetBranches(test.body)
+		assert.Equal(t, test.expected, strings.Join(res, " "))
+	}
+}
+
+func TestParseSingleLineCherryTargetBranches(t *testing.T) {
+	tests := map[string]struct {
+		body     string
+		expected string
+	}{
+		"master": {
+			body:     "cherry-pick to: `master`",
+			expected: "master",
+		},
+		"hosted": {
+			body:     "cherry-pick to: `hosted`",
+			expected: "hosted",
+		},
+		"staging": {
+			body:     "cherry-pick to: `staging`",
+			expected: "staging",
+		},
+		"feature-branch": {
+			body:     "cherry-pick to: `feature-independe_testing-1`",
+			expected: "feature-independe_testing-1",
+		},
+		"1.2.x": {
+			body:     "cherry-pick to: `1.2.x`",
+			expected: "1.2.x",
+		},
+		"1.2.x with escape char": {
+			body:     "cherry-pick to: `1.2.x`" + `\r`,
+			expected: "1.2.x",
+		},
+		"multiple branches": {
+			body:     "cherry-pick to: `master` `hosted` `example-branch`",
```

Comment:
:tada: 

Question:

Will this work if I add, say, a `,` in between the branches?

Reply:
Yes, it will also work if you don't. 

There are also other quirks as well. For instance, this will work:
`` @mender-test-bot `branch1` cherry-pick to: `branch2` ``
Easily fixable by having a regex look-behind, but I thought perhaps no one would ever write something like this. 

Please tell me if you would like the syntax to be "stricter" in those regards, or any other for that matter :)

Reply:
``@mender-test-bot `branch1` cherry-pick to: `branch2` ``will not work anymore now tho